R/prepare_stop_level_data.R

```diff
@@ -26,20 +29,14 @@ granulate_to_stop <- function(mbbs) {
   beep()
 }
 
-#' Processes the stop level information left in the species_comments column of mbbs rows imported from ebird
-#' Returns df of mbbs data that had an informative species_comments broken down to the stop level
-#'@importFrom dplyr %>% filter mutate relocate select
-#'@importFrom stringr str_replace_all str_starts str_split str_extract 
-#'@importFrom tidyr pivot_longer
-#'@param mbbs Any mbbs dataset, either the whole survey area or one county
-process_species_comments <- function(mbbs) {
-  
-  #pull rows that have non-blank species_comments
-  stopsmbbs <- mbbs %>%
-    filter(species_comments != "") 
-  
-  #fix unicode errors on enter signs, and known typos within the dataset
-  stopsmbbs$species_comments <- stopsmbbs$species_comments %>%
+#' Fix species comments
+#' 
+#' Fix unicode errors on enter signs,
+#' and known typos within the dataset
+#' @param x a character vector of species comments
+#' @importFrom stringr str_replace_all
+fix_species_comments <- \(x) {
+    x %>%
     #fix unicode = errors
     str_replace_all("&#61;| =", "=") %>% 
```

Comment:
I think we can make `str_replace_all` call. [Per the docs](https://stringr.tidyverse.org/reference/str_replace.html):

> For str_replace_all() this can also be a named vector (c(pattern1 = replacement1)), in order to perform multiple replacements in each element of string.